blazor/dropdown-list/multicolumn.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Multicolumn in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Multicolumn in Syncfusion Blazor DropDown List component and much more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Multicolumn in Dropdown List 
+
+You can provide two or more columns in the popup by providing the class name `e-multi-column` and the column will be aligned as like grid. [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property is used to get the output as proper alignment.

```

Comment:
Provide two or more columns in the popup by providing the class name `e-multi-column` and the column will be aligned as like grid. The [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property is used to get the output as proper alignment.